### DISCUSSION TITLE
Custom recipe for custom model

### DISCUSSION THREAD

**Discussion by HillTea** [Zrips/CMI#8221] [Enhancement]
Hello guys :)

I have suggestion that i really need please!

![image](https://github.com/Zrips/CMI/assets/102582876/d6b15818-0c3e-4e99-b4c1-0c2afd3bae70)
![image](https://github.com/Zrips/CMI/assets/102582876/243343cd-545d-41c2-a930-a20ffdbe64d1)
![image](https://github.com/Zrips/CMI/assets/102582876/45e42ccf-3f5b-44a1-adc8-db9a02266400)

**Comment by KingTatouille** [comment]
No answer? 😔
